pyriemann/datasets/simulated.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from sklearn.utils.validation import check_random_state
+from .sampling import generate_random_spd_matrix, sample_gaussian_spd
+
```

Comment:
Do you think that it is possible to add a function `make_outliers(n_samples, mean, sigma)`, generating `n_samples` outliers beyond a given distribution of matrices (`mean`, `sigma`)?

See these figures taken from the great repo _Python Outlier Detection_:
https://raw.githubusercontent.com/yzhao062/pyod/master/examples/KNN.png

This function will be useful to test robustness to outliers for tasks like averaging, calibration of artifact detection, etc

Reply:
I like this idea and did a simple implementation of the idea. I am not sure if I should add examples related to it since it is not really on the scope of the PR. Maybe wait for when we'll be adding more examples in a different PR?

Reply:
Thank you so much for this function (and for the whole PR)!
Yes, we could add an example on robust averaging later.

Reply:
For consistency, I think that all functions `make_*` should be in the module `simulated`.